docs/hub/model-card-guidebook.md

```diff
@@ -0,0 +1,19 @@
+
+Our work presents a view of where we think model cards stand right now and where they could go in the future, at Hugging Face and beyond. This work is a “snapshot” of the current state of model cards, informed by a landscape analysis of the many ways ML documentation artefacts have been instantiated. It represents one perspective amongst multiple about both the current state and more aspirational visions of model cards. In this blog post, we summarise our work, including a discussion of the broader, growing landscape of ML documentation tools, the diverse audiences for and opinions about model cards, and potential new templates for model card content. We also explore and develop model cards for machine learning models in the context of the Hugging Face Hub, using the Hub’s features to collaboratively create, discuss, and disseminate model cards for ML models. 
+
+With the launch of this Guidebook, we introduce several new resources and connect together previous work on Model Cards:
+
+1) An updated Model Card template, released earlier [on the Hugging Face Hub](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md), drawing together Model Card work in academia and throughout the industry.
+
+2) An [Annotated version of the updated template](http://github.com/huggingface/hub-docs/docs/hub/model-card-annotated.md), which details how to fill the Card out.
+
+3) A [Model Card Creator Tool](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool), to ease card creation without needing to program, and to help teams share the work of different sections.
+
+4) A [User Study](http://github.com/huggingface/hub-docs/docs/hub/model-cards-user-studies.md) on Model Card usage at Hugging Face
+
+5) A [Landscape Analysis and Literature Review](http://github.com/huggingface/hub-docs/docs/hub/model-card-landscape-analysis.md) of the state of the art in model documentation.
```

Comment:
you should link them using internal links so users can browse them in the /docs/hub website

Reply:
I've assuming e.g:
https://huggingface.co/docs/hub/model-cards/model-card-landscape-analysis

Reply:
or @julien-c by internal you mean using the [doc builder links](https://moon-ci-docs.huggingface.co/docs/hub/pr_545/en/model-card-annotated)